src/validate.ts

```diff
@@ -189,6 +194,9 @@ export function response(compiledPath: CompiledPath | undefined,
   if (response === undefined) {
     response = operation.responses['default'];
   }
-
+  // make Date or ObjectId to string
```

Comment:
@carlansley Test updated. 

I'm using swagger2-koa with koa & mongo, there is some code like ` ctx.body = await Posts.getById(...)`, ctx.body contains Date and ObjectId objects which cause validation failed, and the error message doesn't contain enough detail. 

Actually, this could be adding to swagger2-koa, but swagger2 is an "API-oriented" project, which means the object to validate would eventually being passed to `JSON.stringify()`, if JSON.stringify's return is ok, IMO, the original object should be ok too.